### PR TITLE
update docs for global scss loading

### PR DESCRIPTION
**Pull request by westwick**
per this comment thread: https://github.com/vuejs/vue-loader/issues/328#issuecomment-311579950

This is my first open source contribution :o Let me know if the verbiage is OK, and also not sure how updating other languages works.

### CODE REVIEW

File: docs/en/configurations/pre-processors.md [hunk 1]
```diff
@@ -37,6 +37,29 @@ Contrary to what its name indicates, [*sass*-loader](https://github.com/jtangeld
...
+  {
+    loader: 'sass-resources-loader',
+    options: {
+      resources: path.resolve(__dirname, '../src/style/_variables.scss')
+    }
+  }
+),
+```
+
+It is recommended to only include variables, mixins, etc. in this file, to prevent duplicated css in your final, compiled files.
```
Comment (ralph-tice) [inline docs/en/configurations/pre-processors.md]: wow you're comma happy

File: docs/en/configurations/pre-processors.md [hunk 2]
```diff
@@ -37,6 +37,29 @@ Contrary to what its name indicates, [*sass*-loader](https://github.com/jtangeld
 
 See the [Advanced Loader Configuration](./advanced.md) Section for further information about how to configure vue-loader.
 
+#### Loading a global settings file
+
+A common request is to be able to load a settings file in each component without the need to explicity import it each time, e.g. to use scss variables globally throughout all components. To accomplish this:
```
Comment (ralph-tice) [inline docs/en/configurations/pre-processors.md]: e.g. is a little too cishet you should use esperanto `ekzemple`

File: docs/en/configurations/pre-processors.md [hunk 3]
```diff
@@ -37,6 +37,29 @@ Contrary to what its name indicates, [*sass*-loader](https://github.com/jtangeld
...
+
+``` bash
+npm install sass-resources-loader --save-dev
+```
+
+Then update your `build/utils.js` file:
```
Comment (znck) [inline docs/en/configurations/pre-processors.md]: There won't be a `build/utils.js` file in most projects. Instead, this should be webpack rule, as in L25 above.

Comment (westwick) [inline docs/en/configurations/pre-processors.md]: Let me know if that reads better, I guess I incorrectly assumed most people would be use the webpack template!

**Comment by yyx990803** [comment]
Thanks!

**Pull request merged**